security/file_integrity_operator/file-integrity-operator-release-notes.adoc

```diff
@@ -7,12 +7,30 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-The File Integrity Operator for {product-title} deploys file integrity checking for {op-system} nodes.
+The File Integrity Operator for {product-title} continually runs file integrity checks on {op-system} nodes.
 
 These release notes track the development of the File Integrity Operator in the {product-title}.
 
 For an overview of the File Integrity Operator, see xref:../../security/file_integrity_operator/file-integrity-operator-understanding.adoc#understanding-file-integrity-operator[Understanding the File Integrity Operator].
 
+[id="file-integrity-operator-release-notes-0-1-24"]
+== OpenShift File Integrity Operator 0.1.24
+
+The following advisory is available for the OpenShift File Integrity Operator 0.1.24:
+
+* link:https://access.redhat.com/errata/RHBA-2022:1331[RHBA-2022:1331 OpenShift File Integrity Operator Bug Fix]
+
+[id="file-integrity-operator-0-1-24-new-features-and-enhancements"]
+=== New features and enhancements
+
+* You can now configure the maximum number of backups stored in the `FileIntegrity` Custom Resource (CR) with the `config.maxBackups` attribute. This allows AIDE database and log backups leftover from the `re-init` process to remain on a node. Older backups beyond the configured number are automatically pruned. The default is set to five backups.
+
+[id="openshift-file-integrity-operator-0-1-24-bug-fixes"]
+=== Bug fixes
+* Previously, File Integrity Operator database corruption could occur when updating the File Integrity Operator from older versions. Now, upgrading to the latest version of the File Integrity Operator will complete successfully. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2049206[*BZ#2049206*])
+
+* Previously, `worker-fileintegrity` pods remained in the `Initializing` phase due to `daemonSets` stuck in the `re-init` process. Now,`daemonSets` run to completion and the AIDE database `re-init` process executes successfully. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2072058[*BZ#2072058*])
```

Comment:
Previously when enforcing the default configMap script contents, we compared the wrong data keys. This led to the aide-reinit script not being updated properly on an operator upgrade, and caused the re-init process to break.